src/amount.cpp

```diff
@@ -9,16 +9,18 @@
 
 const std::string CURRENCY_UNIT = "BTC";
 
-CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nSize)
+CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nSize_)
 {
+    int64_t nSize = nSize_;
     if (nSize > 0)
-        nSatoshisPerK = nFeePaid*1000/nSize;
+        nSatoshisPerK = nFeePaid * 1000 / nSize;
```

Comment:
> I guess an alternative would be to change it in the constructor arguments

I thought of that, but didn't mention it at second thought. Before you know it people will want full support for negative sizes as well.